handbook_gen/templates/summary.mako

```diff
@@ -0,0 +1,17 @@
+% for node in summary.children.values():
+${render_node(node, 0)}\
+% endfor
+<%def name="render_node(node, level)">
+<% indent = '' %>\
+% for i in range(level):
+<% indent += '  ' %>\
+% endfor
+% if node.filepath:
+${indent}* [${node.name}](${node.filepath})\
+% else:
+${indent}* ${node.name}\
+% endif
+% for child in node.children.values():
+${render_node(child, level + 1)}\
+% endfor
+</%def>
```

Comment:
Can you add some formatting (indentation) for this template?
And a newline at the end?

Reply:
actually, most likely no ... since doing so will break the output rendering ...